Feature/api format labels

Silentely opened the pull request:
修复API格式显示文本及gemini渠道base url 修改默认显示为 https://generativelanguage.googleapis.com/v1beta
```
openai/chat_completions➡️OpenAI(Chat Completions)
anthropic/messages➡️Anthropic(Messages)
gemini/contents➡️Gemini(Contents)
```

Silentely committed:
fix: update api format labels

Silentely committed:
chore: 添加额外的忽略文件到.gitignore

- 将.pnpm-store目录添加到忽略列表，避免提交pnpm包管理器缓存
- 忽略.serena文件，防止提交本地配置或缓存
- 忽略.claude文件，排除特定开发环境或工具生成的文件

looplj commented on frontend/src/features/channels/components/channels-action-dialog.tsx:
```diff
@@ -27,6 +27,12 @@ import {
 } from '../data/config_providers'
 import { Channel, ChannelType, ApiFormat, createChannelInputSchema, updateChannelInputSchema } from '../data/schema'
 
+const API_FORMAT_TRANSLATION_KEYS: Record<ApiFormat, string> = {
```
这个能不能直接使用 APIFormat 作为 key 的结尾不转一层了，

looplj commented:
感谢 PR。
不过第二点，gemini渠道base url 修改默认显示 base url 修改默认显示为 https://generativelanguage.googleapis.com/v1beta，
这个好像现在就是这样的吧。

Silentely commented:
> 感谢 PR。 不过第二点，gemini渠道base url 修改默认显示 base url 修改默认显示为 [https://generativelanguage.googleapis.com/v1beta，](https://generativelanguage.googleapis.com/v1beta%EF%BC%8C) 这个好像现在就是这样的吧。

我发现是手动删除默认的后框里是显示chat的，还得再改改稍等
<img width="813" height="346" alt="image" src="https://github.com/user-attachments/assets/1c4127cf-d638-4774-ac55-132f93aa6921" />

looplj commented:
好的，了解了，应该是个 bug，placeholder 没有根据具体的 channel type 更新。

翻译帮忙更新一下，括号之前增加一个空格：
OpenAI (Chat Completions)
Anthropic (Messages)
Gemini (Contents)

Silentely committed:
refactor: 重构通道对话框中的 API 格式翻译与占位符逻辑

- 删除硬编码的 `API_FORMAT_TRANSLATION_KEYS` 映射表，动态生成翻译键路径
- 重构 `getApiFormatLabel` 函数，自动将格式字符串（如 'openai/chat_completions'）转换为翻译键（如 'openaiChatCompletions'）
- 修复 `baseURLPlaceholder` 的依赖项，同时监听 `selectedType` 与 `derivedChannelType` 的变化
- 更新中英文翻译文件中 API 格式显示名称的格式，添加空格以提高可读性（例如 "OpenAI (Chat Completions)"）

Silentely commented:
> 好的，了解了，应该是个 bug，placeholder 没有根据具体的 channel type 更新。
> 
> 翻译帮忙更新一下，括号之前增加一个空格： OpenAI (Chat Completions) Anthropic (Messages) Gemini (Contents)

已修改，再看下

looplj commented:
感谢 PR，等我攒几个更新，下个版本发布
